Cargo.toml

```diff
@@ -17,6 +17,7 @@ blocking = ["diesel", "diesel-derive-enum", "dotenv"]
 asynk = ["bb8-postgres",  "postgres-types", "tokio", "async-trait", "typed-builder", "async-recursion"]
 
 [dependencies]
+time = "0.1.44"
```

Comment:
I think this crate is not used.

Reply:
It is used for one for AsyncQueueError

Reply:
delete it in [Error management](https://github.com/ayrat555/fang/pull/67/commits/b3fb2ab6b20328a26099ca8cce2f9f79d38cfc84)